src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Logo from "../logo/Logo";
+import { MenuItems } from "./MenuItems";
+import styles from "./Navbar.module.scss";
+
+// import * as FaIcons from "react-icons/fa";
+
+function Navbar() {
+  return (
+    <nav className={styles.navbar}>
+      <div className={`${styles["navbar__logo-container"]}`}>
+        <Logo classX={`${styles.navbar__logo}`} />
+      </div>
+
+      <ul className={styles.navbar__items}>
+        {MenuItems.map((item, inx) => {
```

Comment:
Avoid using shortcuts in variable names. It makes code harder to read. So no `inx` but `index`.